module/Photo/src/Photo/Service/Admin.php

```diff
@@ -178,7 +178,8 @@ public function upload($files, $album)
             }
         } else {
             throw new \Exception(
-                $translator->translate('The uploaded file is not a valid image')
+                $translator->translate("The uploaded file is not a valid image \nError: " .
+                    json_encode($imageValidator->getMessages()))
```

Comment:
I didn't get the uploader to work. This was added to get more debug information.